manager/integration/tests/test_ha.py

```diff
@@ -2426,3 +2426,94 @@ def test_replica_failure_during_attaching(settings_reset, client, core_api, volu
     del update_disks["extra-disk"]
     node.diskUpdate(disks=update_disks)
     common.wait_for_disk_update(client, node.name, 1)
+
+
+@pytest.mark.skip(reason="TODO") # NOQA
+def test_engine_image_missing_on_some_nodes():
+    """
+    This e2e test follows the manual test steps at:
+    https://github.com/longhorn/longhorn/issues/2081#issuecomment-783747541
+
+    Steps:
+
+    Preparation:
+    1. Set up a backup store
+    2. Let's name the 3 nodes: node-1, node-2, node-3
+
+    Case 1: Test volume operations when engine image DaemonSet is miss
+    scheduled
+    1. Create a volume, vol-1, of 3 replicas
+    2. Create another volume, vol-2, of 3 replicas
+    3. Taint node-1 with the taint: key=value:NoSchedule
+    4. Verify that all functions (attach, snapshot, backup,
+       expand, detach) are working ok for vol-1
+
+    Case 2: Test volume operations when engine image DaemonSet is not fully
```

Comment:
It seems that all cases are in the same context... I mean, does the taint in case1-steo3 still exist when case2 starts? 
If NO, the engine image CR state `deploying` is a transient state, and we cannot validate anything in that period. 
If YES, it's better to emphasize it in the beginning.

Reply:
Thank you. Fixed it to be clear that the subsequent case is continued from the previous case